### PR TITLE
Fix broken timeago helper.

### PR DESCRIPTION
**Pull request by n1k0**
We broke it so now it shows "46 years ago" in the records list. Added tests;

### CODE REVIEW

**Comment by n1k0** [comment]
r=? @leplatrem

File: src/utils.js [hunk 1]
```diff
@@ -19,14 +19,14 @@ export function isObject(thing: any): boolean {
   return typeof thing === "object" && thing !== null && !Array.isArray(thing);
 }
 
-export function timeago(date: string | number): string {
+export function timeago(datesrc: string | number, now: ?Date): string {
   // Show relative time according to current timezone.
-  const nowUTC = (new Date()).getTime();
-  const timestamp = parseInt(date, 10);
+  const nowUTC = (now || new Date()).getTime();
```
Comment (leplatrem) [inline src/utils.js]: Maybe add a comment to explicit that it's used for testing?

File: test/utils_test.js [hunk 2]
```diff
@@ -201,3 +202,19 @@ describe("buildAttachmentUrl", () => {
...
 });
+
+describe("timeago", function() {
+  it("should convert a timestamp", () => {
+    expect(timeago(new Date().getTime() - 86400000)).eql("1 day ago");
+  });
+
+  it("should convert a date string", () => {
+    expect(timeago("2016-12-01T00:00:00.000Z",
+           new Date("2016-12-02T00:00:00.000Z"))).eql("1 day ago");
```
Comment (leplatrem) [inline test/utils_test.js]: I looked in `CollectionRecords` and maybe we don't have to support passing strings. I think we only want to support millisecond timestamps.

Comment (n1k0) [inline test/utils_test.js]: I discovered we had date strings passed upon inspection (it's in `get lastModified()` where we do `const date = new Date(lastModified).toISOString();`)

**Comment by n1k0** [comment]
I'll definitely remove support for date strings, what a freaking mess...

File: src/components/collection/CollectionRecords.js [hunk 3]
```diff
@@ -27,9 +27,9 @@ class Row extends Component {
     if (!lastModified) {
       return null;
     }
-    const date = new Date(lastModified).toISOString();
-    return (
-      <span title={date}>{timeago(date)}</span>
+    const date = new Date(lastModified);
+    return Number.isNaN(date.getTime()) ? null : (
```
Comment (n1k0) [inline src/components/collection/CollectionRecords.js]: `date.toJSON()` seems to return `null` on invalid dates, I'll use that instead.

**Comment by n1k0** [comment]
It seems travis has stopped running our tests on this projectn, because of a pip issue:

**Comment by Natim** [comment]
I know how to fix that, remove the `whoami` in .travis.yml: `pip install --user `whoami` virtualenv`

**Pull request merged**